MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieSearchBar+Design.swift

```diff
@@ -0,0 +1,139 @@
+import UIKit
+import SnapKit
+
+extension MovieSearchBar: DesignProtocol {
+    
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        searchGrayFieldView = UIView()
+        addSubview(searchGrayFieldView)
+        
+        let image = ImageEnum.searchBarIcon.image
+        searchBarIconImageView = UIImageView(image: image)
```

Comment:
You can just write `UIImageView(image: ImageEnum.searchBarIcon.image)`

Reply:
I changed it [here](https://github.com/lihter/Movie-App/pull/24/files#diff-6a7e0c3cc088b3a980b8a92c08ad5d190f909d67f14aa13f150bdc992ab52302R16).